commands/quote.js

```diff
@@ -3,20 +3,24 @@ const fetch = require('node-fetch');
 
 exports.run = async (message) => {
 const { quote, author } = await fetch('http://quotes.stormconsultancy.co.uk/random.json').then(response => response.json())
-const { file } = await fetch('https://aws.random.cat/meow').then(response => response.json())
+const { contents } = await fetch('https://quotes.rest/qod.json').then(response => response.json())
+const { data } = await fetch('https://api.giphy.com/v1/gifs/random?api_key=20SloZAgXiC07Zqu9pBwIllwoVJ9v1xn').then(response => response.json())
 const { url } = await fetch('https://api.nasa.gov/planetary/apod?api_key=DEMO_KEY').then(response => response.json())
```

Comment:
Can you please move the environment variable to the `.env` file? We already have the dotenv package pulled in, so once you add `GIPHY_API_KEY=` to the .env file, you can access the value in your code with `process.env.GIPHY_API_KEY`. Thanks!

Reply:
Nice tips. Feel free to review my new commit when you can. Thanks!